vermouth/data/force_fields/martini3001/modifications.ff

```diff
@@ -25,3 +25,26 @@ BB {"replace": {"atype": "P6", "charge": 0}}
 NH2-ter
 [ atoms ]
 BB {"replace": {"atype": "P6", "charge": 0}}
+
+[ modification ]
+CCAP-ter
+[ atoms ]
+BB {"replace": {"atype": "P1", "charge": 0}}
+
+[ modification ]
+NCAP-ter
+[ atoms ]
+BB {"replace": {"atype": "P2", "charge": 0}}
+
+[ modification ]
+HIS_PROT
+[ atoms ]
+BB  {"resname": "HIS", "replace": {"resname": "HIP"}}
+SC1 {"resname": "HIS", "replace": {"resname": "HIP"}}
+SC2 {"resname": "HIS", "replace": {"atype": "TP1dq", "charge": "0.5", "resname": "HIP"}}
+SC3 {"resname": "HIS", "replace": {"atype": "TP1dq", "charge": "0.5", "resname": "HIP"}}
+[ edges ]
+BB SC1 
+SC1 SC2
+SC1 SC3
+SC2 SC3
```

Comment:
This looks a little fishy. Is it a leftover test case?

Reply:
if you don't have the edges an error is raised from the connected graph check for modifications

Reply:
let's keep it for now. we need to change these things later anyways. but this modification can be a paper example